apps/web/src/server/applications/applications.controller.js

```diff
@@ -19,9 +19,10 @@ import * as applicationsService from './applications.service.js';
  */
 export async function viewDashboard({ params }, res) {
 	const { domainType } = params;
-	const applications = await applicationsService.findOpenApplicationsByDomainType(domainType);
+	const allApplications = await applicationsService.findOpenApplicationsByDomainType(domainType);
+	const readyApplications = allApplications.filter((application) => application.status !== 'draft');
```

Comment:
this should fix https://pins-ds.atlassian.net/browse/BOAS-232 - will assign it to you @lutamKC so we can close it when this is merged